### PR TITLE
Docker: pass extra arguments to scylla

### PR DESCRIPTION
**Pull request by dkropachev**
Currently there is no way to pass scylla arguments from docker-entrypoint to scylla,
but time to time it is needed.
Example: https://github.com/scylladb/scylla-operator/issues/177

### CODE REVIEW

File: dist/docker/redhat/scyllasetup.py [hunk 1]
```diff
@@ -151,5 +152,22 @@ def arguments(self):
 
         args += ["--blocked-reactor-notify-ms 999999999"]
 
+        argument = None
+        for arg in self._extra_args:
```
Comment (penberg) [inline dist/docker/redhat/scyllasetup.py:156]: Is this transformation really needed? Can't we just append `extra_arguments` to `args`?

**Review by penberg — COMMENTED** [review]
Nice improvement!

File: dist/docker/redhat/scyllasetup.py (hunk 1 above)
Comment (dkropachev) [reply dist/docker/redhat/scyllasetup.py]: In base cases we can drop it in favor of simple .split .

Tested with: 
--new_arg1=val --new_arg2 val --new_arg3="val" --new_arg4 "val" --new_arg5
Result:
SCYLLA_DOCKER_ARGS="--overprovisioned --listen-address 127.0.1.1 --rpc-address 127.0.1.1 --seed-provider-parameters seeds=127.0.1.1 --blocked-reactor-notify-ms 999999999 --new_arg1=val --new_arg2 val --new_arg3=val --new_arg4 val --new_arg5"

**Comment by penberg** [comment]
Queued to `next`. Please send a follow up to document this in `docs/docker-hub.md`.

**Comment by dkropachev** [comment]
> Queued to `next`. Please send a follow up to document this in `docs/docker-hub.md`.

Pushed another commit to put it into the docs

**Comment by mmatczuk** [comment]
This is a move in a good direction, we also need a way to specify which services to run i.e. how do I run scylla only the entrypoint cannot be changed.

**Comment by avikivity** [comment]
Suppose someone makes a spelling error in an argument. Now, parse_known_args() will ignore them and they will be passed to scylla. Scylla will start, fail, and be restarted by the supervisor.

**Pull request closed without merging**